contexts/CookingStepsContext.js

```diff
@@ -17,7 +17,27 @@ export function CookingStepsProvider({ children }) {
     0
   );
 
-  const stepList = determineStepList(recipesSelection);
+  let stepList = [];
```

Comment:
As I am not sure if this is the correct way to go in this specific situation, I leave you just a hint to think about if you really need a single = or if it happened by mistake and should be a triple ===

Reply:
Thanks Teresa for code review. I am using = because this is an assignment. If I'm not mistaken, you use the === in a comparison.
I use "let" because the contents of stepList change again in the course of the code and that's why I couldn't use a constant :)